index.js

```diff
@@ -67,6 +68,30 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener("click", function(event){
+    toggleColor(event.target)
+})
+
 /***** Deliverable 2 *****/
+const likeButton = document.querySelector(".like-button")
+
+likeButton.addEventListener("click", function(event) {
+    // traveler.likes += 1
+    likes.textContent = `${traveler.likes += 1} Likes`
+})
+
 
```

Comment:
Good, but what if there were multiple travelers who all had likes and I needed to find only the one that I clicked on? Here's one way we could approach that..

```
const likeBtn = document.querySelector(".like-button");
function addNewLike(e){
    const likesPTag = e.target.previousElementSibling;
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```
With ES6 notation
```
const likeBtn = document.querySelector(".like-button");
const addNewLike = e => {
    const likesPTag = e.target.previousElementSibling;
    let currentLikes = parseInt(likesPTag.innerText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}
likeBtn.addEventListener("click", addNewLike)
```